Sources/Sentry/SentryHttpTransport.m

```diff
@@ -417,11 +434,7 @@ - (void)finishedSending
 {
     @synchronized(self) {
         self.isSending = NO;
```

Comment:
since `stopFlushing` also sets `_isFlushing = NO`, does it make sense to do it here as well? If we want it to be set before the `@synchronized` barrier, maybe we also set it before that in `stopFlushing` on line 243?

Reply:
I'm unsure what you exactly mean. We don't want it to be set before the `@synchronized` barrier, but we want to update both `isSending` and `isFlushing` in the same synchronized block, which we already do here. We call `@synchronized` twice here, which is OK because `@synchronized` is reentrant. I think the change is legit. I'm going to follow up with another PR after clarifying this comment, if required.

Reply:
I think I read `isSending` here and my brain interpreted it as `isFlushing` 🤦🏻